samples/debug/ppi_trace/src/main.c

```diff
@@ -10,18 +10,19 @@
 #include <hal/nrf_rtc.h>
 #include <hal/nrf_clock.h>
 #include <device.h>
+#include <devicetree.h>
 #include <logging/log.h>
 
 LOG_MODULE_REGISTER(app);
 
 #define ALARM_PERIOD_US 50000
 
-#if IS_ENABLED(CONFIG_USE_RTC2)
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(rtc2), okay)
```

Comment:
Since the `USE_RTC2` option is to stay, I'd rather drop this change, for consistency.

Reply:
done